_saas-integrations/shopify/v1/shopify-v1.md

```diff
@@ -66,6 +66,9 @@ feature-summary: |
 #      Setup Instructions    #
 # -------------------------- #
 
+requirements-info: |
```

Comment:
This isn't displaying on the preview page - did you test it?

Hint: Take a look at the `singer.html` layout to see why.